content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Configure_the_Primary_YubiHSM_2_Device.adoc

```diff
@@ -159,6 +159,7 @@ The wrap key thereby generated is saved to the HSM 2 device.
 
 *Step 8*: When the relevant prompt is displayed, each of the three wrap key custodians should take their turn in front of the screen to record their share.
 
+
```

Comment:
nit maybe, should we really add an extra blank line here? I don't think it changes anything in the rendering.

Reply:
You are right. There should not be an extra blank line there. I am rather particular about having two blank lines before a new heading, but this note belongs to step 8.